src/components/v2/shared/DistributionSplitsSection/DistributionSplitCard.tsx

```diff
@@ -0,0 +1,221 @@
+              <div
+                style={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'space-between',
+                }}
+              >
+                <span style={{ cursor: 'pointer' }}>
+                  <FormattedAddress address={split.beneficiary} />
+                </span>
+                {isOwner && <CrownFilled />}
+              </div>
+            </Col>
+          </Row>
+        )}
+
+        {parseInt(split.projectId ?? '0') > 0 ? (
+          <Row>
+            <Col span={labelColSpan}>
+              <label>
+                <Trans>Token beneficiary:</Trans>
+              </label>
+            </Col>
+            <Col span={dataColSpan}>
+              <span style={{ cursor: 'pointer' }}>
+                <FormattedAddress address={split.beneficiary} />
+              </span>
+            </Col>
+          </Row>
+        ) : null}
+
+        <Row gutter={gutter} style={{ width: '100%' }} align="middle">
+          <Col span={labelColSpan}>
+            <label>Percentage:</label>
+          </Col>
+          <Col span={dataColSpan}>
+            <div
+              style={{
+                display: 'flex',
+                width: '100%',
+                justifyContent: 'space-between',
+                alignItems: 'center',
+              }}
+            >
+              <span
+                style={{
+                  marginRight: 10,
+                  width: 100,
+                  maxWidth: 100,
+                }}
+              >
+                <Space size="large">
+                  <span>
+                    {formatSplitPercent(BigNumber.from(split.percent))}%
+                  </span>
+                  {!parseWad(distributionLimit).eq(MAX_DISTRIBUTION_LIMIT) && (
+                    <span>
+                      <CurrencySymbol currency={currencyName} />
+                      {getDistributionAmountFromPercentBeforeFee({
+                        percent: parseFloat(
+                          formatSplitPercent(BigNumber.from(split.percent)),
+                        ),
+                        distributionLimit,
+                      })}
+                    </span>
+                  )}
+                </Space>
+              </span>
+            </div>
+          </Col>
+        </Row>
+
+        {split.lockedUntil ? (
+          <Row gutter={gutter} style={{ width: '100%' }} align="middle">
+            <Col span={labelColSpan}>
+              <label>
+                <Trans>Locked:</Trans>
+              </label>
+            </Col>
+            <Col span={dataColSpan}>
+              until {formatDate((split.lockedUntil ?? 0) * 1000, 'yyyy-MM-DD')}
+            </Col>
+          </Row>
+        ) : null}
+      </Space>
+
+      {isLocked ? (
+        <LockOutlined style={{ color: colors.icon.disabled }} />
+      ) : (
+        <Button
+          type="text"
+          onClick={e => {
+            onSplitsChanged([
```

Comment:
**suggestion**: I don't think this component should know anything about `splits`: I think there's a solution where we might:
- add an `onDelete` prop that takes a function
- Call `onDelete()` here
- in the parent component, handle `onDelete` with this same logic below

Spreading logic across components is something we want to avoid for maintainability.

Reply:
Yeah there are other reasons we needs splits to be passed down though, mainly validating for 'same address already added' in the payout modal. So i figured not worth another another `onDelete` prop if we need `splits`